torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -51,7 +51,7 @@ class FSDPCommContext:
 
     def lazy_init(self, device: torch.device):
         self.device_handle = _get_device_handle(device.type)
-        if device.type not in ["cuda", "hpu"]:
+        if device.type not in ["cuda", "hpu", torch._C._get_privateuse1_backend_name()]:
```

Comment:
Do we have an API to detect if the given device supports streams? @albanD @FFFrog

Reply:
@albanD @FFFrog thoughts?

Reply:
Sorry for the late comments.
As far as I know, there is no some one API detecting if supports stream.

This PR can be closed now due to https://github.com/pytorch/pytorch/pull/139539

Reply:
Thanks for the info, I’m gonna close this PR.